script/precompile_tasks

```diff
@@ -1,13 +1,9 @@
-#! /bin/bash
-
-# Exit if any subcommand fails
-set -e
-set -o pipefail
+#!/bin/sh -e
```

Comment:
I assume this is doing more or less what the above two lines are doing, which is ensuring that if a line (or part of a line) fails, it will exit. I'm surprised `set -e` doesn't work as it seems to be in the POSIX standard (I believe [this](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/sh.html) is the relevant link for the flags) but it this works everywhere then I'm fine with it. 🙂

Reply:
Oh, and I guess switching from `bash` to `sh`

Reply:
We can do both yes, I was changing the shebang so I added the option there.
Did you found that `set -e` and `sh -e` are different, can you point me the line?

Reply:
From what I read, I think you're right that they are the same. I was asking because I wanted to know if I was missing something. You had mentioned POSIX compliance and I read that these are both compliant so I wanted to make sure I wasn't missing anything. Thanks for the confirmation 😄